pkg/commands/archive_test.go

```diff
@@ -64,13 +64,13 @@ func testArchiveSource(t *testing.T, tc obtainTestCase) {
 	}
 	assert.Nil(t, err)
 
-	var projectDockerComposeFile string
+	var projectDockerFile string
 	if tc.targetDir == "" {
-		projectDockerComposeFile = "docker-compose.yml"
+		projectDockerFile = "Dockerfile"
 	} else {
-		projectDockerComposeFile = filepath.Join(tc.targetDir, "docker-compose.yml")
+		projectDockerFile = filepath.Join(tc.targetDir, "Dockerfile")
```

Comment:
for `filepath.Join` we should use https://github.com/cyphar/filepath-securejoin - it's not really a big deal but worth keeping it as a reminder somewhere